deliverables/submitted/requirements.md

```diff
@@ -1,32 +1,242 @@
+
+Postconditions: Price information is displayed
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.004.jpeg)
+
+Use Case: Price Analysis
+
+Actor: Trend Analyst
+
+Description: Trend Analyst wants to compare prices of a product over the last 30 days to understand whether this product group may be collectively dropping or rising recently. Preconditions: Product page is open on desired product.
+
+Main Flow: Making graphs filtered for 30 days, collapsing all results to one graph. Postconditions: Observing rising or falling prices.
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.005.png)
+
+Use Case: Compare Prices
+
+Actor: Shopper
+
+Description: The shopper will be able to view prices for a given product from multiple different sources.
+
+Preconditions: Shopper will search for a product and view price information
+
+Main Flow: Seeing multiple prices from different sellers
+
+Postconditions: Observing which sellers are selling products for cheaper/more expensive prices.
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.006.jpeg)
+
+Use Case: See price analysis
+
+Actor: Shopper
+
+Description: Shopper will enter the name of a product into a search bar, and then see a value of the prices showing past prices.
+
+Preconditions: Web Page must be open.
+
+Main Flow: Searching
+
+Postconditions: Finding the product prices
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.007.jpeg)
+
+![](Aspose.Words.2dfcf8d4-4087-4a8f-9f9f-da974c5bf25d.008.jpeg)
+
+7. **User stories**
+
+Write two user stories for each member of the group. They can be related to the same features described in the use cases or to different ones. Adopt the following format: "As a <ROLE>, I want <SOMETHING> so that <GOAL>."
+
+Establish a priority level for each user story and estimate how many hours each one will demand using the planning poker approach.
+
+1. As an online shopper, I want an easy-to-use tool so that I can get the best prices on the things I purchase.
+1. As a mother of four, I want a product that allows me to quickly look for the best deal on any given item to save time and money.
+1. As a college student, I would love a tool that would show me the cheapest price for a product with one simple search.
+1. As a deal seeker, I want a tool that lets me see the best times to shop for a product.
+1. As a shopper, I want to be informed about the cost of a product between multiple vendors so I can spend less.
+1. As a PC builder, I want to know if the price of an expensive part is trending down so I can make educated economic decisions.
+1. As a chronic gamer, I want to know the prices for new and old video games so that I spend the right amount of money on a game.
+1. As a single dad, I want a set of legos so my son will like me more, but I’m so I need the best price.
+1. As a McDonald's manager, I want new chairs for my restaurant so that customers will be happy in the establishment.
```

Comment:
this seems more like a scenario of use. Remember that this is a specification of a requirement. Someone will receive this "requirement" to implement in the system. Please, modify